### DISCUSSION TITLE
Hebrew support?

### DISCUSSION THREAD

**Discussion by afarlie** [psb1558/Junicode-font#267] [Ideas]
Prompted by -[ Index:Gesenius' Hebrew Grammar (1910 Kautzsch-Cowley edition).djvu](https://en.wikisource.org/wiki/Index:Gesenius%27_Hebrew_Grammar_(1910_Kautzsch-Cowley_edition).djvu) . Not something I could contribute on directly, but in places the Hebrew font used in GHG, and the Hebrew font provided with wikimedia aren't quite the same.  Alternatively are there 'free' fonts I could suggest to Wikisource contributors that are a closer match?

**Comment by psb1558** [comment]
I'm very far from expert on Hebrew, but Google Fonts lists 54 fonts with Hebrew support. Perhaps browse there for something suitable?
